CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ If you are looking for the Ledger Releasing and Versioning Process then you can
 [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
 ## 8.7
-
+	
+- Changed the type of lenses ppCommitteeMaxTermLengthL, ppuCommitteeMaxTermLengthL, ppMaxBBSizeL, ppMaxTxSizeL, ppMaxBHSizeL, ppuMaxBBSizeL, ppuMaxTxSizeL and ppuMaxBHSizeL
```

Comment:
That is not the changelog that we need to change. Changelog for the package need to be adjusted.

```suggestion
```

Reply:
done